examples/advanced/sklearn-svm/README.md

```diff
@@ -79,6 +79,44 @@ Available arguments:
 - `--kernel`: Kernel type - linear, poly, rbf, or sigmoid (default: rbf)
 - `--backend`: Backend library - sklearn or cuml (default: sklearn)
 - `--data_path`: Path to cancer CSV file (default: /tmp/nvflare/dataset/cancer.csv)
+- `--split_method`: Data split method - 'uniform' or 'custom' (default: uniform)
+
+### Per-Client Data Splits
+
+By default (`--split_method uniform`), all clients use the same data ranges, which is suitable for testing on small datasets like Breast Cancer.
+
+For realistic federated learning with non-overlapping data, use `--split_method custom`:
+
+```bash
+python job.py --n_clients 3 --kernel rbf --split_method custom
+```
+
+This will:
+- Calculate non-overlapping training data ranges for each client
+- Use 80% of data (455 samples) for training, split among clients
+- Last 20% (114 samples) used as shared validation set
+- Pass different `--train_start`, `--train_end`, `--valid_start`, `--valid_end` arguments to each client
```

Comment:
"As an alternative, we can also save the splitted data to three individual files, and pass different "--data_path" to each client"